x-pack/plugins/security_solution/public/onboarding/components/onboarding_body/cards/common/card_content_panel.tsx

```diff
@@ -5,13 +5,23 @@
  * 2.0.
  */
 import React, { type PropsWithChildren } from 'react';
-import { EuiPanel, type EuiPanelProps } from '@elastic/eui';
+import { COLOR_MODES_STANDARD, EuiPanel, useEuiTheme, type EuiPanelProps } from '@elastic/eui';
+import { css } from '@emotion/css';
 
 export const OnboardingCardContentPanel = React.memo<PropsWithChildren<EuiPanelProps>>(
   ({ children, ...panelProps }) => {
+    const { euiTheme, colorMode } = useEuiTheme();
+    const isDarkMode = colorMode === COLOR_MODES_STANDARD.dark;
     return (
-      <EuiPanel hasShadow={false} hasBorder={false} paddingSize="m">
-        <EuiPanel {...panelProps} hasShadow={false} paddingSize="l">
+      <EuiPanel
+        className={css`
+          background-color: ${isDarkMode && euiTheme.colors.lightestShade};
+          padding: 8px 16px 16px 16px;
+        `}
+        hasShadow={false}
+        hasBorder={false}
+      >
+        <EuiPanel {...panelProps} hasShadow={false}>
```

Comment:
Angela did the same change, but I think we only need to remove the top padding, the `paddingSize="l"` should be okay.
```suggestion
        <EuiPanel {...panelProps} hasShadow={false} paddingSize="l" style={{ paddingTop: 0 }}>
```